README.md

```diff
@@ -2,27 +2,24 @@
 
 Course materials for Research Software Engineering course.
 
-Current plan for structure (as advertised):
+Current plan for structure:
 
 Research Software Engineering
 
-21st, 28th Feb and 7th, 14th, 21st March 2014             2:00pm - 5:00pm
-
 Content:
 In this course, you will move beyond programming, to learn how to construct reliable, readable, efficient research software in a collaborative environment. The emphasis is on practical techniques, tips, and technologies to effectively build and maintain complex code. This is a short (15 hours over 5 half-days), intensive, practical course. The content of each of the 5 half-day units is as follows:
 
-1. Code management. Collaborating around code. Distributed version control. Git. Github. Issue tracking. Code review and pull requests. Branches and merging. Software licenses. Citing software. Software sustainability.
-1. Testing scientific software. Unit testing, regression testing. Test driven design. Expectations and assertions. Mocking. Build-and-test servers. Negative testing. Sensible error messages. Managed logging. Debugging and debuggers. Coverage measurement.
+1. Code management. Collaborating around code. Distributed version control. Git. Github. Issue tracking. Code review and pull requests. Branches and merging. Continuous Integration. Citing software. Software sustainability.
+1. Testing scientific software. Sensible error messages. Error handling. Print-debugging. Debuggers. Test pyramid: unit testing, regression testing. Test driven design. Expectations and assertions. Mocking. Negative testing.
 1. Best practice in construction. Comments. Coding conventions. Documentation. Refactoring. IDEs. Configuration files. Using libraries.
 1. Design and development. Object oriented design. Software as engineering. Pragmatic use of diagram languages. Requirements engineering. Agile and Waterfall. Functional and architectural design.
-1. Building and deployment. Build tools. CMake and Scons.
-   Engineering for portability. Packaging. Deploying.
+1. Building and deployment. Build tools. Packaging. Deploying. Open Source community and interaction.
```

Comment:
focus on pip here? Is there any benefit in covering conda as well? What build tools where you thinking of here?

Reply:
Ah, I could have removed "Build tools" maybe, though definitely want to cover automated setup "there should be one command to get from cloned repo to able to use/contribute".

Conda: I've seen it's popular among machine learning folks so I'm in favour of either covering it or finding out how biased my sample of machine learning folks is.